src/utils.js

```diff
@@ -473,7 +472,14 @@ export default class Utils {
                 }
                 return true;
             } else {
-                this.xhr(u, b, m, a);
+                if ('fetch' in window.parent && typeof window.parent.fetch === 'function') {
+                    const controller = new AbortController();
```

Comment:
I think it is also necessary to check whether `AbortController` exists because Fetch API and `AbortController` is not always exists at the same time in some versions of browsers.

Reply:
good point. maybe only for Safari 12 but 10% of internet users.